config/fraudbusters-mgmt/values.yaml.gotmpl

```
@@ -89,6 +89,33 @@ metrics:
         path: /actuator/prometheus
         scheme: http
 
+ingress:
```

Тут забыты 2 строчки 
```suggestion
{{ $domainWithNamespace := printf "%s.%s" .Release.Namespace .Values.services.ingress.rootDomain }}
{{ $ingressDomain := .Values.services.ingress.namespacedDomain | ternary $domainWithNamespace .Values.services.ingress.rootDomain }}

ingress:
```